modules/osd-create-cluster-ccs.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_create/creating-an-aws-cluster.adoc
+// * osd_cluster_create/creating-a-gcp-cluster.adoc
+
+ifeval::["{context}" == "osd-creating-a-cluster-on-aws"]
+:osd-on-aws:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-gcp"]
+:osd-on-gcp:
+endif::[]
+
+:_content-type: PROCEDURE
+ifdef::osd-on-aws[]
+[id="osd-create-aws-cluster-ccs_{context}"]
+= Creating a cluster on AWS with CCS
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+[id="osd-create-gcp-cluster-ccs_{context}"]
+= Creating a cluster on GCP with CCS
+endif::osd-on-gcp[]
+
+By using the Customer Cloud Subscription (CCS) billing model, you can create an {product-title} cluster in an existing 
+ifdef::osd-on-aws[]
+{AWS} 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+{GCP} 
+endif::osd-on-gcp[]
+account that you own.
+
+You must meet several prerequisites if you use the CCS model to deploy and manage {product-title} into your AWS account.
+
+.Prerequisites
+
+ifdef::osd-on-aws[]
+* You have configured your AWS account for use with {product-title}.
+* You have not deployed any services in your AWS account.
+* You have configured the AWS account quotas and limits that are required to support the desired cluster size.
+* You have an `osdCcsAdmin` AWS Identity and Access Management (IAM) user with the `AdministratorAccess` policy attached.
+* You have set up a service control policy (SCP) in your AWS organization.
+* Consider having *Business Support* or higher from AWS.
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+* You have configured your GCP account for use with {product-title}.
+* You have configured the GCP account quotas and limits that are required to support the desired cluster size.
+* You have created a GCP project.
++
+[NOTE]
+====
+The project name must be 10 characters or less.
+====
+* You have enabled the Google Cloud Resource Manager API in your GCP project. For more information about enabling APIs for your project, see link:https://cloud.google.com/endpoints/docs/openapi/enable-api[the Google Cloud documentation].
+* You have an IAM service account in GCP called `osd-ccs-admin` with the following roles attached:
+  ** DNS Administrator
+  ** Organization Policy Viewer Owner
```

Comment:
The prerequisite is different with UI, would you please double confirm?

Reply:
Following on from a discussion in Slack with our Engineering team, a review of the details in https://issues.redhat.com/browse/OSDOCS-3194 and my own tests to deploy OSD on GCP with CCS, I have now updated these prerequisites to match what is in the UI.